src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -29,6 +29,11 @@ object HeaderPattern {
   val hashLineComment = """(?s)((?:#[^\n\r]*(?:\n|\r|\r\n))+(?:\n|\r|\r\n)+)(.*)""".r
 }
 
+object HeaderKeys {
```

Comment:
Please move to `HeaderPlugin`, so that unqualified access in `build.sbt` is possible. Also rename to `HeaderKey` (singular). Thanks!